marlowe-apps/example-processor.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Read the input.
+INPUT=$(cat)
+
+# Extract the name of the choice.
+SYMBOL=$(echo "$INPUT" | jq -r .symbol)
+
+# Extract the minimum bound.
+MIN=$(echo "$INPUT" | jq -r .bounds[0].from)
+
+# Extract the maximum bound.
+MAX=$(echo "$INPUT" | jq -r .bounds[0].to)
+
+if [[ "$SYMBOL" == "RANDOM.ORG" ]]
+then
+  # Fetch the data.
+  curl 'https://www.random.org/integers/?format=plain&col=1&rnd=new&base=10&num=1&min='"$MIN"'&max='"$MAX"
```

Comment:
Any reason for doing an external curl instead of using the `$RANDOM` variable?

https://linuxhint.com/generate-random-number-bash/

Reply:
Yes, http://www.random.org/ provides true random numbers based on atmospheric turbulence or quantum processes, but `RANDOM` just provides *pseudo*-random numbers.